entity-framework/core/what-is-new/ef-core-2.1.md

```diff
@@ -0,0 +1,71 @@
+
+- Storing enums as strings
+- Mapping unsigned integers with SQL Server
+- Automatic encryption and decryption of property values
+
+Read the [section on value conversions](xref:core/modeling/value-conversions) for more information.  
+
+## LINQ GroupBy translation
+Before EF Core 2.1, the GroupBy LINQ operator would always be evaluated in memory. We now support translating it to the SQL GROUP BY clause in most common cases.
+
+## Data Seeding
+With the new release it will be possible to provide initial data to populate a database. Unlike in EF6, in EF Core, seeding data is associated to an entity type as part of the model configuration. Then EF Core migrations can automatically compute what insert, update or delete operations need to be applied when upgrading the database to a new version of the model.
+
+As an example, you can use this to configure seed data for a Post in `OnModelCreating`:
+
+``` csharp
+modelBuilder.Entity<Post>().SeedData(new Post{ Id = 1, Text = "Hello World!" });
+```
+
+## Query types
+An EF Core model can now include query types. Unlike entity types, query types do not have keys defined on them and cannot be inserted, deleted or updated (i.e. they are read-only), but they can be returned directly by queries. Some of the usage scenarios for query types are:
+
+- Mapping to views without primary keys
+- Mapping to tables without primary keys
+- Mapping to queries defined in the model
+- Serving as the return type for FromSql() queries
+
+## Include for derived types
+It will be now possible to specify navigation properties only defined on derived types when writing expressions for the `Include` method. For the strongly typed version of `Include`, we support using either an explicit cast or the `as` operator. We also now support referencing the names of navigation property defined on derived types in the string version of `Include`:
+
+``` csharp
+var option1 = context.People.Include(p => ((Student)p).School);
+var option2 = context.People.Include(p => (p as Student).School);
+var option3 = context.People.Include("School");
+```
+
+## System.Transactions support
+We have added the ability to work with System.Transactions features such as TransactionScope. This will work on both .NET Framework and .NET Core when using database providers that support it.
+
+## Better column ordering in initial migration
+Based on customer feedback, we have updated migrations to initially generate columns for tables in the same order as properties are declared in classes. Note that we cannot change order when new members are added after the initial table creation.
+
+## Optimization of correlated subqueries
```

Comment:
@maumar does this look better? Can you provide a good example?

Reply:
Good example (if its not too complicated) could be: 
context.Customers.Select(c => c.Orders.Where(o => o.Id > 10).Select(o => o.Name).ToList())

Maybe add that you opt-in by adding ToList()/ToArray() call to the correlated query?